studio/schemas/documents/course.js

```diff
@@ -126,9 +126,31 @@ export default {
         list: [
           {title: 'Free', value: 'free'},
           {title: 'Pro', value: 'pro'},
+          {
+            title: 'Community Resource',
+            value: 'community-resource',
+            description: 'Free Forever',
+          },
```

Comment:
Moved Community Resource into the access level field. The reasoning behind that is that community resources are a special case of free where they are considered free forever. It's also impossible for a resource to both be PRO and a community resource at the same time